.github/workflows/pr-akka-samples.yml

```diff
@@ -88,17 +89,24 @@ jobs:
           cd akka-samples-${{ matrix.sample }}
           git config user.name "lightbend-tools-secure"
           git config user.email "lightbend-tools-secure@lightbend.com"
-          BRANCH=update-sdk-$sdk_version
+          # include timestamp to make the branch name unique
+          timestamp=`date '+%Y%m%d-%H%M%S'`
+          BRANCH=update-sdk-$sdk_version-$timestamp
           git checkout -b $BRANCH
           rm -rf ./src/*
           rsync -a ../docs/build/bundle/${{ matrix.sample }}/ ./
           rm -f .bundleignore
-          git add -f .idea/runConfigurations/Run_locally.xml
-          git add .
-          git commit . -m "chore: bump SDK version to $sdk_version"
-          git push --set-upstream origin $BRANCH
-          gh pr create -B main -t "Auto PR - Bump SDK version to $sdk_version" \
+          
+          if [ -n "$(git status --porcelain)" ]; then
+            git add -f .idea/runConfigurations/Run_locally.xml
+            git add .
+            git commit . -m "chore: bump SDK version to $sdk_version"
+            git push --set-upstream origin $BRANCH
+            gh pr create -B main -t "Auto PR - Bump SDK version to $sdk_version" \
               -b "This PR should update SDK (pom.xml) and latest source code.  [$GITHUB_SHA]($COMMIT_URL$GITHUB_SHA)"
+          else
+            echo "no changes";
```

Comment:
Since not failing anymore, better to print the version from the `./docs/bin/version.sh` and also `cat pom.xml`.

At least we will have some tracing in case of issues.